### ISSUE TITLE
VS Code installer for linux blocks key pressed events while it is open

### ISSUE THREAD

**Issue by IustinMicu** [microsoft/vscode#132877]
Does this issue occur when all extensions are disabled?: Yes

There is a bug in the VS Code installer for linux (at least in this one: code_1.60.0-1630494279_amd64.deb).

After starting the installer I let it do it's thing in the background. I later discovered that it was blocking keyboard events. When I pressed the "Done" button and it was closed my keyboard started working again.

1. I suppose the installer listens to the key pressed events and then stops propagating them.
2. Why is the installer listening to keyboard events in the first place? Especially when it runs minimized.

**Comment by joaomoreno** [comment]
> VS Code installer

Can you show me a screenshot of this installer window?

**Comment by IustinMicu** [comment]
I gave you the full version number of the installer. It should be enough to try to reproduce the issue.
Have you tried to install it? If so did you manage to reproduce it?
You are now asking me go through the trouble of reinstalling it so I can provide a screenshot. A screenshot of what? You didnt even tell me what you want to see in the screenshot or at what moment of the installation to take the screenshot.

Anyways I updated the VS Code using this installer: code_1.60.2-1632313585_amd64.deb and the issue didn't reproduce anymore. I suppose it was somehow fixed so this bug can be closed.
